src/Pages/FarmTImeLine.js

```diff
@@ -79,118 +106,144 @@ function FarmTimeline() {
+        <Grid container className={classes.root} spacing={2} >
+          <Grid item xs={12}>
+            <Grid container justify="center">
+
+            </Grid>
+            <Grid container justify="center" spacing={2}>
+              <List component="nav" aria-label="secondary mailbox folders">
+                <ListItem>
+                  <span
+                    style={{
+                      backgroundColor: "white",
+                      border: "1px solid gray",
+                      padding: "5px",
+                      margin: "5px",
+                      color: "gray",
+                      fontSize: "20px",
                     }}
-                  />
-                </Grid>
-              </MuiPickersUtilsProvider>
-            </ListItem>
-
-            <ListItem key="5">
-              <Button variant="outlined" color="primary" onClick={handleCLick}>
-                SHOW
-              </Button>
-            </ListItem>
-          </List>
-        </Grid>
-      </Grid>
-    </Grid>
+                  >
+                    {" "}
+              FIELD TIMELINE
+            </span>
+                </ListItem>
+                <ListItem key="1">
+                  <FormControl className={classes.formControl}>
+                    <InputLabel id="locality-label">Locality</InputLabel>
+                    <Select
+                      id="locality"
+                      name="locality"
+                      value={locality}
+                      onChange={handleChange}
+                    >
+                      {locationsDataFromServer.map((locationDataFromServer) => {
+                        return (
+                          <MenuItem
+                            key={locationDataFromServer.code}
+                            value={locationDataFromServer.code}
+                          >
+                            {locationDataFromServer.displayStr}
+                          </MenuItem>
+                        );
+                      })}
+                    </Select>
+                  </FormControl>
+                </ListItem>
+
+                <ListItem key="2">
+                  <FormControl className={classes.formControl}>
+                    <InputLabel id="fId-label">Field ID</InputLabel>
+                    <Select
+                      id="fieldId"
+                      name="fieldId"
+                      value={fieldId}
+                      onChange={handleChange}
+                    >
+                      {fields.map((field) => {
+                        return (
+                          <MenuItem key={field.identifier} value={field.identifier}>
+                            {field.identifier}
+                          </MenuItem>
+                        );
+                      })}
+                    </Select>
+                  </FormControl>
+                </ListItem>
+
+                <ListItem>
+                  <MuiPickersUtilsProvider utils={DateFnsUtils}>
```

Comment:
You don't need to use `<MuiPickersUtilsProvider utils={DateFnsUtils}>` two times for the two date fields. Can move it to a higher level, which wraps both the date pickers.